### PR TITLE
Consistency in the confirmation modals

### LINKED ISSUE #1731: Consistency in the confirmation modals
https://github.com/bigbinary/neeto-engineering-ui-ux/issues/147

cc: @praveen-murali-ind

### PR DESCRIPTION
**Pull request by nitinprojects**
- Fixes #1731 

**Description**  : Please refer before/.after snapshots below

Screenshot 2023-06-19 at 5 38 11 PM

---------

-------------

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
~~- [ ] I have verified the functionality in some of the neeto web-apps.~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added the necessary label (patch/minor/major - If package publish~~
      is required).

**Reviewers**

### CODE REVIEW

**Comment by nitinprojects** [comment]
@praveen-murali-ind _a Please review.

File: stories/Overlays/Alert.stories.jsx [hunk 1]
```diff
@@ -78,25 +79,28 @@ export const Sizes = () => {
           </div>
           <Alert
             isOpen={showAlertSmall}
-            message="Leaving this page will delete all unsaved changes."
+            message="All of your unsaved changes will be lost. This can't be undone"
```
Comment (praveen-murali-ind) [inline stories/Overlays/Alert.stories.jsx:82]: @nitinprojects _A Please add a period at the end of the message.

File: stories/Overlays/Alert.stories.jsx [hunk 2]
```diff
@@ -78,25 +79,28 @@ export const Sizes = () => {
...
-            title="Leave page with unsaved changes?"
+            submitButtonLabel="Discard changes"
+            title="You have unsaved changes"
             onClose={() => setShowAlertSmall(false)}
             onSubmit={() => setShowAlertSmall(false)}
           />
           <Alert
             isOpen={showAlertMedium}
-            message="Leaving this page will delete all unsaved changes."
+            message="All of your unsaved changes will be lost. This can't be undone"
```
Comment (praveen-murali-ind) [inline stories/Overlays/Alert.stories.jsx:91]: @nitinprojects Same here. Period missing.

File: stories/Overlays/Alert.stories.jsx [hunk 3]
```diff
@@ -78,25 +79,28 @@ export const Sizes = () => {
...
-            title="Leave page with unsaved changes?"
+            submitButtonLabel="Discard changes"
+            title="You have unsaved changes"
             onClose={() => setShowAlertMedium(false)}
             onSubmit={() => setShowAlertMedium(false)}
           />
           <Alert
             isOpen={showAlertLarge}
-            message="Leaving this page will delete all unsaved changes."
+            message="All of your unsaved changes will be lost. This can't be undone"
```
Comment (praveen-murali-ind) [inline stories/Overlays/Alert.stories.jsx:100]: @nitinprojects Period missing.

**Comment by praveen-murali-ind** [comment]
@nitinprojects _a Periods are missing in 3 examples. The rest of the changes look good to me. Once done please assign the PR to @amaldinesh7 for further review. 

Screenshot 2023-06-20 at 10 51 37 AM

**Comment by nitinprojects** [comment]
@amaldinesh7 _a Please review.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**